key.txt

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..FuF41AU2mYHtBerE.WhFnjl3LliqVHb5hW4HTc_ms4YbXMJJ1T-58E-Qvg-xsQ2aj-2trgfqfa3aZFLBotUojnwS9yg5KK9oGdTb5w7HUToBPsEudTXk0OKsmLA5f3xDYP9SBjy7UfkDYNxvjZ8yVP1dDBCXciqbSX2PDzqWCqzbwTFHMKnROjAtzgwieAgKV_06IinxsF_EwvD-FiP3OrM9_n8h3Hz4suvcokjgqN-WEqH0TjQL1dqpfhnCU3TooVxa6zV2e0iTxJ4_aRP_Eiw0KRljywY68JnO5Y5tvHXvVfgIFEtrDqRYLlqETM5dpyvxQf41jx8h6QpSD84rbNSDJK0KLbT9i4JyAwVbbUIPriudd8j78I-Pd6K8SyclvZdcZs-KbdNuuREbYxX_f-6YDV7IMg0pUKxI5F0Kjf2WCc1DW1trrRG2KIm7y4MUJT3Fr5vxg_LgXM0wgzSklUpWOrsEJDkkWRrEVg-yyeqW_66_FB-5NBL0fommAsYx7NENq8yTYEn5m__NQENv5kCQT8rXKJDe-sXbORLiuJEUEKxAj6po3c5HxWdp8MVfZBeWS9AwpCPFgc2PsxSIxWBDQpaxn05ncGNo8EPE0HKAxKcrlytIGQiosBWB6arowd9hp9IOu9PKxdyBtRp7zihs6DntRnFHQpBsBSKv2QNzY4gYl-8p9HNtx3aPwLPHyWDdvqtzjSXRGAA1q4iJMxGAx-34PWg1YMTbKujgZGAwa8G7f1ka-BZGohXKWX88m6YsqQPDYnCU89PDBFetVHAJRbwAtE-PQejWa6svKIhPIIAYMRORITmhe0FZDWHmsDJQENXkh7Ut6o2nCzhy3fDlVqNeVnLHFHP4c3agfMPfeMFhzwnOte-cYkPbjbp2WWIURmJckdJOIc3nT3N23HJiuR7c93WEa97pEjM5YQ3zVKcVbiyTWHGgkCcB0vNmNpXqx2fizwxh3Slwx1ipzNhbJ1drtPVxlGD3SAV_vze1ibpymdbjr09xCHvCKF_rRSMDR3Kii7mnsCrZXQok8f090vdAa6K7LtedWsBeZyssvYxREsz3NuBBwKgh4BIgbJwfKBexFNOn0My755oBK6WuIcVboxxUGSHY6DgyT7o3ehXu2bcquLOtpHh-NwLr3kfd_BztKtQF9GjLyvVIUPNAAyjbq0knJdn4G3pFBfOqeVtImV9Ej-De03Qjbd3P1QIYvWzbxsZDJJ7GR0L4eIkwx6o9wh006SYRvHXMMb5AtDdxAxCobQgqrCnbZru2mlJOqgsw7Yrugg30tS1kBq9MJOM0m1MgXEnpWkLdtKT88NhqPzXjeZZPLJyXRarYeYchi_jze75sq2sx-4Ab_TNDzn_i6aAGK0u5tw_Tn7yJZxjz07qTTjRJLQD_kx-RQCAqdFi6y4U0j3pdE2k2XqM1BVLnUsHlEAE0JAOhkzFkddhGnzBqHQskjf54fpd-v0BD_duZLMvcqxVd9skNn1U1OF_MWQsVZL40El-0QaLEa4_B5feTuEbERn0UqKZJxJeXTzjuw5xapFNjXjlqmmZWGMx6rzHOaYvTC_mrQ6G4dkhqM8YS5L2gm9VS_8NqP8lJ-NX1vulbVcDCBYGDU3l3fmoQto6nI7qZzZq3wq3HZG8g6Vn7ctcZKGAjAQjQob0AHKLZwkxZMQsB36H5_ZUzdHJERlwNZx_ma6q_frgRxekx_ahhMPu5pL8YM-NFSxq5yhJx9d6e6Z5e_-vCw4gMscKwlkUXYVnzkaE_FvlZd5yBOY0CAxcIf-jfskcQUi9MksLb-zM311nbQxQ1fy81BMyb1BNTJsrjELzxQtqwe-hoDKBbts3GyCQCZO2mFFTQ8lisqV-n8UqSdh91GE1wa7Is5VEpGXRDbuOMLY7-mi79XVJfJS0ty2jjZ5NVmHJ3tZ25Gw0VDOtgH4Yye95NebZSrDe6-JI8ZP-uGsWSv3h09-0ZUC9R3P6kNjMutXBca3F8Vd65QP1A6-jMRtN0GH3Hp93BEybdTVafKfEh1XH7LEapYX1IwGBFVGUXm-I8z3xxa0oIYG7TETVxSkIoC_uXschuUTVoK9Z0dvS5e9lYJaE47IHh8tCe4gqbn-gYuPykTKx5b3LTiVoySTfZ2LlixV_z8GgUpH7PuuHIjj6OSsecwONftR6jag7dW5xjBJm91kIHsAJ1i7XA6WVD1RNN80smMDN5g5anfWreYAT5cvPny7Pn6x8ENFPDH6SXKPtEQCZw-xm8l04f_Z0U-W5dBLqJHcZE.SM-VE3ZHLk6PijbFLCdK2w
```

Comment:
Is there a need to keep it created? If so, maybe it would be better to keep it empty, instead of keeping there something that look alike a valid key?

Reply:
There is no need at all to have this file. It's funny, I had added a tip on readme.md + an exception on .gitignore to avoid exactly this problem. 

I added a more general exception on .gitignore instead and removed the tip on readme.me:
- *Tip: Use a file named .session_key in chatgpt-api top directory. It will be ignored by git - see .gitignore.*

There is a test failing, but it is not related to the content of this merge request. ChatGPT is currently using CloudFare DDoS protection and the requests might have been blocked or should be done differently... Just a quick assessment, didn't look into the details.